oeps/processes/oep-0063-proc-toc-arbitration.rst

```diff
@@ -0,0 +1,115 @@
+
+Abstract
+********
+
+The Technical Oversight Committee (TOC) aims to empower the project's contributors to take decisions. The person closest to the work will often take the best decisions, and empowered contributors face less friction. 
+
+However, some of the decisions can sometimes be complex: lack of consensus, decisions with long-lasting reprecussions. This can result in delaying taking *any* decision, which can be more hurtful than any of the choices.
+
+In this OEP, the TOC defines a process for community members to bring up topics to the board's attention, and request to take a decision about a proposal (an "Arbitration"). 
+
+Motivation
+**********
+
+The TOC discussed during the April 2023 meeting a `series of guidelines for operating and taking decisions <https://discuss.openedx.org/t/2023-04-11-toc-meeting-summary/10019#governance-norms-5>`_. This includes establishing a way for the community to use the TOC to help take decisions, which this OEP implements.
+
+Guidelines
+==========
+
+From the `TOC meeting notes <https://discuss.openedx.org/t/2023-04-11-toc-meeting-summary/10019#governance-norms-5>`_:
+
+*The TOC should not operate “in the weeds.” We don’t have the time to be involved day-to-day technical decisions, and those closest to the work will make the best decisions. The TOC should not act as a “stage gate.” We should not create friction for contributors unnecessarily, but should:*
+   
+- *Guide the project toward shared goals*
+- *Help refine and improve plans and designs*
+- *Prevent the project from making bad decisions that are hard to reverse.*
+   
+*The TOC should weigh in:*
+   
+- *on “one-way-doors,” decisions that are hard to reverse, where choices are mutually exclusive*
+- *when the community is having difficulty reaching consensus*
+- *when an important decision is being deferred and not making a call is causing friction*
+- *to focus the community to collaborate on key, shared value, say, investment in maturing LTI*
+
+Specification
+*************
+
+- Requests for arbitration can be posted by anyone in the community - but need to be supported by at least two core contributors or a TOC member
+- The TOC members are encouraged to participate in community discussions early on, before the TOC is asked to arbitrate.
+
+Step-by-step - How to submit an arbitration
+===========================================
+
+- Open a discussion in a public place, such as the `forum <discuss.openedx.org/>`_ or using one of the formal decision recording formats such as `OEPs <https://open-edx-proposals.readthedocs.io/en/latest/>`_. The topics presented need to have been discussed in the community before being formally considered by the TOC.
+- After public review, post a formal arbitration request to the TOC. This can be done in a forum post or a `github ticket <https://github.com/openedx/wg-coordination/issues/new>`_ mentioning the TOC chair (@e0d). See the format below.
```

Comment:
Will @e0d always be the TOC chair? I am wary of including info that's likely to change (and forget to be updated) in a foundational document such as an OEP

Reply:
@sarina Good point. It's useful to be able to give the exact username of the chair for the ping here, but we can replace it with a link to the wiki, which names the TOC chair: https://openedx.atlassian.net/wiki/spaces/COMM/pages/3575939113/Technical+Oversight+Committee+TOC#Details . I have added @e0d 's github and forum usernames there, this way this remains easily available.